### PR TITLE
OPENINSAI: SWI-4546 - Prepare sairoute.py file for upstreaming

### PR DESCRIPTION
**Pull request by pkochmanska**

### CODE REVIEW

File: ptf/sairoute.py [hunk 1]
```diff
@@ -0,0 +1,1118 @@
...
+        neighbor_entry = sai_thrift_neighbor_entry_t(
+            rif_id=self.port10_rif, ip_address=sai_ipaddress('10.10.10.2'))
+        sai_thrift_create_neighbor_entry(self.client, neighbor_entry,
+                                         dst_mac_address=dmac)
+
+        route_entry = sai_thrift_route_entry_t(
+            vr_id=self.default_vrf, destination=sai_ipprefix('10.10.10.1/32'))
+        sai_thrift_create_route_entry(
+            self.client, route_entry, next_hop_id=nhop,
+            packet_action=SAI_PACKET_ACTION_TRAP)
```
Comment (ravi861) [inline ptf/sairoute.py]: This is probably something we are not supporting or some hack. The test says dropRouteTest but the packet action here is TRAP. Please check.

File: ptf/sairoute.py [hunk 2]
```diff
@@ -0,0 +1,1118 @@
...
+            sai_thrift_remove_neighbor_entry(self.client, neighbor_entry2)
+            sai_thrift_remove_neighbor_entry(self.client, neighbor_entry1)
+
+            sai_thrift_remove_next_hop(self.client, nhop3)
+            sai_thrift_remove_next_hop(self.client, nhop2)
+            sai_thrift_remove_next_hop(self.client, nhop1)
+
+            sai_thrift_remove_router_interface(self.client, vlan100_rif)
+            sai_thrift_set_port_attribute(self.client, self.port24,
+                                          port_vlan_id=1)
```
Comment (ravi861) [inline ptf/sairoute.py]: Please replace all instances in tearDown where we reset port_vlan_id=1 to port_vlan_id=0

File: ptf/sairoute.py (hunk 1 above)
Comment (pkochmanska) [reply ptf/sairoute.py]: Yes, it was a workaroud because we don't support DROP action in route_entry, I'll fix it.

**Review by ravi861 — APPROVED** [review]
Approved for upstream

**Pull request closed without merging**